<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>191</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background-color: rgb(255, 184, 184);
}</string>
    </property>
    <property name="title">
     <string>Кнопки анимаций</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="bntDown">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Движение вниз</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntRight">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Движение вправо</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntHandUp">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Поднять руку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntHonor">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Отдать честь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntStart">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Движение +  рука</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntHpS">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Движение + честь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bntRect">
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color: green;
}</string>
       </property>
       <property name="text">
        <string>Движение - родитель</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>420</y>
      <width>781</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
background-color: yellow;
}</string>
    </property>
    <property name="title">
     <string>Данные</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>90</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: green;
}</string>
     </property>
     <property name="text">
      <string>Закрыть</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>514</width>
       <height>88</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Лабораторная работа номер 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Задание: Нарисовать солдатика, перемещающегося и отдающего честь.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Родительский класс – перемещающийся прямоугольник.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Выполнил студент группы 353505 Суровцев Артём</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>0</y>
       <width>631</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>После нажатия любой из кнопок солдат вернется в начальное положение!!!</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bnt_prikol">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>60</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
background-color: green;
}</string>
     </property>
     <property name="text">
      <string>Прикол</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
